extensions/amp-twitter/0.1/amp-twitter.js

```diff
@@ -62,34 +93,67 @@ class AmpTwitter extends AMP.BaseElement {
     const iframe = getIframe(this.win, this.element, 'twitter', null,
         {allowFullscreen: true});
     this.applyFillContent(iframe);
-    listenFor(iframe, 'embed-size', data => {
-      // We only get the message if and when there is a tweet to display,
-      // so hide the placeholder
-      this.togglePlaceholder(false);
-      this./*OK*/changeHeight(data['height']);
+    iframe.classList.add('i-amp-html-twitter-iframe');
```

Comment:
is this use? (if so, `s/amp-html/amphtml`)

Reply:
Oops, fixed.